<launch>
  <!-- <node pkg="rosbag" type="play" name="rosbag" required="true" args="$(find image_view)/test.bag"/> -->
  <node name="extract" pkg="image_view" type="extract_images" respawn="false" required="true" output="screen" cwd="ROS_HOME">
    <remap from="image" to="/camera/color/image_raw"/>
  </node>
  <node name="republish" pkg="image_transport" type="republish" respawn="false" required="true" output="screen" args="compressed in:=/camera/color/image_raw raw out:=/camera/color/image_raw">
  </node>
</launch>